src/widgets/page/Page.m.css

```diff
@@ -12,14 +12,13 @@
 
 .content {
 	flex-grow: 1;
-	min-width: 900px;
-	width: 50%;
+	width: 1000px;
 	margin: 0 auto 80px;
 }
 
 .footer {
 	color: #999 !important;
```

Comment:
I know this isn't your code but is there a reason we are using a hex variable and `!important` ?